eng/scripts/Show-FailureLogs.ps1

```diff
@@ -6,7 +6,8 @@
 # sensitive information.
 
 $logFiles = Get-ChildItem -Recurse -Filter *.log
-$filteredLogs = $logFiles.Where({ $_.Name -in ('vcpkg-bootstrap.log', 'vcpkg-manifest-install.log') })
+$vcpkgLogFileNames = ('vcpkg-bootstrap.log', 'vcpkg-manifest-install.log')
```

Comment:
Doesn't really matter here, but FYI powershell array syntax is either:
- `<item>,<item>,<item>`
- @(<item>, <item>, <item>)

So what's happening here is the parentheses are just resolving to the first array syntax and are unnecessary.